doc/source/whatsnew/v1.0.0.rst

```diff
@@ -525,7 +525,7 @@ Use :meth:`arrays.IntegerArray.to_numpy` with an explicit ``na_value`` instead.
 
    a.to_numpy(dtype="float", na_value=np.nan)
 
-**Reductions can return ``pd.NA``**
+**Reductions can return ``pd.NA``\ **
```

Comment:
I'm not sure if monospace texts can be bold to. Maybe the next is a simpler solution?

```suggestion
**Reductions can return** ``pd.NA``
```

I'm assuming the backslash space tells sphinx that there is nothing separating those two formatting options, tight?

Reply:
Yes backslash-space is "nothing" in RST.

Reply:
But I used your proposition, it's cleaner to read as plain text.